### PR TITLE
Poetry

### PR DESCRIPTION
**Pull request by henrikstranneheim**
This PR adds/fixes:

- Adds a poetry lock file and poetry toml file
- Enables demux to be installed by pip or poetry
- Update readme to reflect this
- Added functionality to use bumpversion to also bump version in poetry toml file instead of setup.py

**How to prepare for test**:
- Nothing to test

**Review:**
- [ ] code approved by
- [ ] "Merge" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by barrystokman — APPROVED** [review]
Moving demultiplexing into the future!

:100:

**Comment by henrikstranneheim** [comment]
@barrystokman Should I merge this or do you want to test some python parts on clinical-db?

**Comment by barrystokman** [comment]
Did you mean thalamus? I'm not sure what to do on clinical-db.

I'm fine with merging this, I don't think there will be much deployment and things on thalamus from now on.

File: README.md [hunk 1]
```diff
@@ -57,7 +63,7 @@ username: apiuser
 password: somepassword
 ```
 
-Files will be blacked automatically with each push to github. If you would like to automatically [Black][black-url] format your commits on your local machince:
+Files will be blacked automatically with each push to github. If you would like to automatically [Black][black-url] format your commits on your local machine:
```
Comment (karlnyr) [inline README.md:66]: pc here would be "Black is automatically applied to files with each push to GitHub"

**Comment by henrikstranneheim** [comment]
> Did you mean thalamus? I'm not sure what to do on clinical-db.
> 

Yeas, sorry I meant Thalamus - do many servers....

**Review by karlnyr — COMMENTED** [review]
Looks great! The sooner the merge the earlier I can make audits to my work as well! 💯

**Comment by henrikstranneheim** [comment]
👍 Alright merging!

**Pull request merged**